### PR TITLE
[DPG-731] added adapter master data service

### PR DESCRIPTION
**Pull request by shailesh-egov**

### CODE REVIEW

File: deploy-as-code/helm/environments/ifix-qa.yaml [hunk 1]
```diff
@@ -62,6 +62,7 @@ cluster-configs:
         fiscal-event-service: "http://fiscal-event-service.ifix:8080/"
         ifix-zuul: "http://ifix-zuul.ifix:8080/"
         ifix-department-entity-service: "http://ifix-department-entity-service.mgramseva:8080/"
+        adapter-master-data-service: "http://adapter-master-data-service.mgramseva:8080/"
```
Comment (rushang7) [inline deploy-as-code/helm/environments/ifix-qa.yaml]: Are you deploying this in `mgramseva` namespace or `ifix`?

File: deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Common Labels
+labels:
+  app: "adapter-master-data-service"
+  group: "rainmaker"
```
Comment (rushang7) [inline deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml]: This isn't rainmaker project. Name it `ifix`.

File: deploy-as-code/helm/charts/core-services/egov-mdms-service/values.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ ingress:
 initContainers:
   gitSync:
     enabled: true
-    repo: "git@github.com:egovernments/egov-mdms-data"
+    repo: "git@github.com:misdwss/mdms-mgramseva"
```
Comment (rushang7) [inline deploy-as-code/helm/charts/core-services/egov-mdms-service/values.yaml]: Raise a separate PR for this change. This isn't related to adapter master data.

File: deploy-as-code/helm/environments/ifix-qa.yaml (hunk 1 above)
Comment (shailesh-egov) [reply deploy-as-code/helm/environments/ifix-qa.yaml]: ifix

File: deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml (hunk 2 above)
Comment (shailesh-egov) [reply deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml]: Changed

File: deploy-as-code/helm/charts/core-services/egov-mdms-service/values.yaml (hunk 3 above)
Comment (shailesh-egov) [reply deploy-as-code/helm/charts/core-services/egov-mdms-service/values.yaml]: changed

File: deploy-as-code/helm/environments/ifix-qa.yaml (hunk 1 above)
Comment (shailesh-egov) [reply deploy-as-code/helm/environments/ifix-qa.yaml]: Updated namespace.

**Comment by shailesh-egov** [comment]
not needed, we are not moving mgramseva services on ifix.

**Pull request closed without merging**